### PR TITLE
Use absolute storage blob uri if found in requestUri

### PR DESCRIPTION
**Pull request by chintanrp**
Use absolute storage blob uri if found in requestUri else generate it based on storageaccount and cloud environment endpoints

### CODE REVIEW

**Review by CraigWiand — COMMENTED** [review]
Look reasonable to me.  It is possible to create a test for this?

File: pyServer/AzureDiskInspectService.py [hunk 1]
```diff
@@ -262,7 +262,12 @@ def ParseUrlArguments(self, urlPath, sasKey):
             container_blob_name = container_blob_name + '/' + urlSplit[urlSplitIndex]
             urlSplitIndex = urlSplitIndex + 1
         
-        storageUrl = urllib.parse.urlunparse(
+        self.telemetryLogger.info("Storage Account from requestUri: " + storageAcctName)
+        if ".blob." in storageAcctName:
```
Comment (waynekuo) [inline pyServer/AzureDiskInspectService.py:266]: This looks good. Consider adding a small comment here on the two different storage account parsing logic.

**Review by gitjpzh — COMMENTED** [review]
Looks good to me.

File: pyServer/AzureDiskInspectService.py (hunk 1 above)
Comment (chintanrp) [reply pyServer/AzureDiskInspectService.py:267]: Comment added

**Comment by chintanrp** [comment]
> Look reasonable to me. It is possible to create a test for this?

Added new test which include full storage url in Request uri

**Review by waynekuo — APPROVED** [review]

**Pull request merged**